### ISSUE TITLE
[Bug][A11Y][Demo App] With "Orange Inverse" theme, some text color is not reversed

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#953]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS18.5

### App version

0.19.0

### App build number

1757539222

### App build type

beta

### Describe the issue

With "Orange Inverse" theme, the text used to explain each component is not reversed, black on black 

<img width="300" height="200" alt="" src="https://github.com/user-attachments/assets/c8bf601f-384e-4ff0-8434-d03edba8457b" />

### Expected Behavior

inverse color also on texts

### Accessibility settings

none

**Comment by B3nz01d** [comment]
Not needed anymore
